install-dependencies.sh

```
@@ -0,0 +1,121 @@
+    cd ..
+else
+    echo -e "${RED}❌ Backend directory not found!${NC}"
+    exit 1
+fi
+
+# Install Frontend Dependencies
+echo -e "\n${YELLOW}🎨 Installing Frontend Dependencies...${NC}"
+echo -e "${BLUE}📂 Navigating to frontend directory...${NC}"
+
+if [ -d "frontend" ]; then
+    cd frontend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)